monitor/flowgen/bridging/bridging.go

```diff
@@ -31,7 +32,7 @@ func NewJob(
 	}
 
 	return types.Job{
-		Name:    "Bridging",
+		Name:    fmt.Sprintf("Bridging (%v->%v)", srcChain, dstChain),
```

Comment:
avoid using ints for chains, as humans struggle to read numbers, chain names make things a LOT easier to debug/read/understand and saves a lot of time.